GitUI/CommandsDialogs/FormCommit.cs

```diff
@@ -1116,10 +1116,8 @@ private void RestoreSelectedFiles(IReadOnlyList<GitItemStatus> unstagedFiles, IR
             {
                 _currentFilesList.SelectedItems = newSelection;
             }
-            else
-            {
-                SelectStoredNextIndex();
-            }
+
+            SelectStoredNextIndex();
```

Comment:
This seem to change the Unstaged selection, if there is still a valid unstaged file, selection should not be changed.

Reply:
Could you give an example? I'm afraid I'm not following you.